### PR TITLE
ryu/ryu/app/network_awareness/shortest_forwarding.py

### PR DESCRIPTION
**Pull request by Hypnotised1**
[PATH]10.0.0.6<-->10.0.0.30: [3002, 2002, 1003, 2008, 3008]
[PATH]10.0.0.6<-->10.0.0.30: [1003, 2008, 3008]
[PATH]10.0.0.6<-->10.0.0.30: [3008]
There are some error path print on the screen. So, I changed some code to improve it.

[shortest_forwarding.txt](https://github.com/osrg/ryu/files/368937/shortest_forwarding.txt)

### CODE REVIEW

**Comment by iwaseyusuke** [comment]
Which files exactly do you want commit?
It seems to be including so many unnecessary files.

**Comment by iwaseyusuke** [comment]
I'm sorry I didn't explain it enough.

Your request seems to have unnecessary (e.g. some patches seems to be fixed or deleted later) patches,
so I think you need to cleanup and merge them, first.
Only the files under ryu/app/network_awareness/ seems to be needed in this case, right?

And according to [CONTRIBUTING.rst](https://github.com/osrg/ryu/blob/master/CONTRIBUTING.rst#submitting-a-change), Ryu project does not use Pull Request on GitHub.
How about sending your patches to ryu-devel@lists.sourceforge.net?

File: ryu/app/network_awareness/network_awareness.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+from ryu.lib.packet import packet
+from ryu.lib.packet import ethernet
+from ryu.lib.packet import ipv4
+from ryu.lib.packet import arp
+from ryu.lib import hub
+
+from ryu.topology import event, switches
+from ryu.topology.api import get_switch, get_link
+import setting
+
```
Comment (ShohrehHps) [inline ryu/app/network_awareness/network_awareness.py]: import

**Comment by anarkiwi** [comment]
Closing as past 12 months old. Please open a new PR if required.

**Pull request closed without merging**